api/src/middlewares/services/utils/getPost.ts

```diff
@@ -1,5 +1,11 @@
 import * as additionalContexts from "./additionalContexts.json";
+import { FormType } from "../../../types/FormDataBody";
+import { MARRIAGE_FORM_TYPES } from "../../../utils/formTypes";
 
-export function getPost(country: string, post?: string) {
-  return post ?? additionalContexts.countries?.[country]?.post;
+export function getPost(country: string, type: FormType, post?: string) {
```

Comment:
I think refactor this to `getPostForMarriage` and `getPostForCertifyCopy`, then we won't need to add parameters in other places.